K2Bridge.Tests.UnitTests/JsonConverters/AggregationConverterTests.cs

```diff
@@ -73,143 +73,150 @@ public class AggregationConverterTests
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new CardinalityAggregation {
-                            FieldAlias = "_2",
-                            FieldName = "metric",
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new AverageAggregation
+                    {
+                        Field = "metric",
+                        Key = "2",
                     },
+                    SubAggregations = new AggregationDictionary(),
                 },
+            },
         };
 
-        private static readonly Aggregation ExpectedValidAvgAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedNoFieldsAvgAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new AvgAggregation {
-                            FieldAlias = "_2",
-                            FieldName = "metric",
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new AverageAggregation
+                    {
+                        Field = null,
+                        Key = "2",
                     },
+                    SubAggregations = new AggregationDictionary(),
                 },
+            },
         };
 
-        private static readonly Aggregation ExpectedNoFieldsAvgAggregation = new Aggregation()
+        private static readonly AggregationContainer ExpectedValidCardinalityAggregation = new AggregationContainer()
         {
             PrimaryAggregation = null,
-            SubAggregations = new Dictionary<string, Aggregation>
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
                 {
-                    { "2", new Aggregation() {
-                        PrimaryAggregation = new AvgAggregation {
-                            FieldAlias = "_2",
-                            FieldName = null,
-                            },
-                        SubAggregations = new Dictionary<string, Aggregation>(),
-                        }
+                    PrimaryAggregation = new CardinalityAggregation
+                    {
+                        Field = "metric",
```

Comment:
should be null?

Reply:
ok now I see that the order of tests were changed. can you change the order to reduce diff changes

Reply:
Yes. Good point. Order has been changed to default one.